addons/project/views/rating_rating_views.xml

```diff
@@ -133,6 +133,10 @@
             </xpath>
             <xpath expr="//filter[@name='resource']" position="attributes">
                 <attribute name="string">Task</attribute>
+                <attribute name="invisible">0</attribute>
+            </xpath>
+            <xpath expr="//filter[@name='month']" position="attributes">
+                <attribute name="string">Submitted on</attribute>
```

Comment:
I feel like this could maybe be a generic change? Not sure

Reply:
Yes we can make it a generic change.
It seems to be great idea :+1: